api-reference/v1.0/toc.yml

```
@@ -5151,6 +5153,20 @@ items:
                   href: api/ipnamedlocation-update.md
                 - name: Delete
                   href: api/ipnamedlocation-delete.md
+          - name: Authentication context class references
```

Maybe "Authentication context" instead - to reference the capability instead of the resource name?
```suggestion
          - name: Authentication context
```